sdk/va-report-components/documentation/docs/api/setFormatterLocale.md

```diff
@@ -0,0 +1,30 @@
+---
+id: setFormatterLocale
+title: setFormatterLocale
+---
+
+```
+setFormatterLocale(locale): void
+```
+
+`setFormatterLocale` enables the user to set a language to be used for formatting numeric values and dates. If not specified this will fallback to either the language specified by calling setLocale, or the language specified in the browser settings.
+
+## Arguments
+
+### `locale: string | null`
+
+The locale should be specified as a language code and optional country code. This is the same format used by [navigator.language](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language)
```

Comment:
Small tweaks to help with L10N: ... and **an** optional country code. This is the same format **that is** used by...

Reply:
If this needs that change, then we also need that change for the existing doc for setLocale, https://developer.sas.com/sdk/va/docs/api/setLocale/

Reply:
@timerikson does this actually take `null`?  i ask because `setLocale` is documented to only take `string`.

Reply:
It does.. setLocale also takes null. I should update that.  Internally we do not take null for setLocale because something at a higher level will set it to be the browser language if the user did not specify anything. But even internally we take null for formatterLocale because that means it should fall back to locale. At the VA-SDK level passing null to setLocale will cause it to use the browser locale.

Reply:
@brmorr, it looks like I missed those text changes when we reviewed setLocale.

Reply:
Done. Thanks.